packages/postgres-database/src/simcore_postgres_database/models/funcapi_function_job_collections_table.py

```diff
@@ -0,0 +1,51 @@
+"""Functions table
```

Comment:
THOUGHT: generally I am minimalistic with doc and put the effort in self-documented code. I would remove all these comments

Reply:
Removed